doc/Customization-guide/Code-style/Java/index.md

```diff
@@ -4,4 +4,39 @@ title: Java
 permalink: /:path/
 sort_idx: 10
 ---
-Specific per-language code style guidelines. Check style configuration files for Eclipse and IDEA. Style check also must be automated in the build process (after code styles are defined here).
+
+* TOC
+{:toc}
+
+We follow [Google Java Style](https://google.github.io/styleguide/javaguide.html) in Kaa Java SDK.  
```

Comment:
Whole Java source code of Kaa project must conform to the [Google Java Style](https://google.github.io/styleguide/javaguide.html).